### PR TITLE
Profile pagev2

### PR DESCRIPTION
**Pull request by ghost**
Add package react player, add style for profile page and demoPage with flexbox

### CODE REVIEW

**Review by LauraCourquin — DISMISSED** [review]

File: frontend/src/components/CardProfileInfo/index.jsx [hunk 1]
```diff
@@ -1,36 +1,84 @@
...
+      />
       <div className="profileInfo">
+        <div
+          className={
+            artistData.avatar === null || artistData.avatar === ""
+              ? "defaultAvatar"
+              : "avatarImg"
+          }
+        />
+        {/* <div className="avatarImg"> </div> */}
```
Comment (LuigiMenez) [inline frontend/src/components/CardProfileInfo/index.jsx]: No dead code

File: frontend/src/components/CardProfileInfo/index.jsx [hunk 2]
```diff
@@ -1,36 +1,84 @@
...
+        <div className="description">
+          <a href="www.google.com"> + d'information </a>{" "}
+        </div>
+
         <div>
-          <button type="button">follow</button>
-          <img src={agenda} alt="" />
-          <img src={insta} alt="" />
-          <img src={linkedin} alt="" />
+          <img className="imgAgenda" src={agenda} alt="" />
```
Comment (LuigiMenez) [inline frontend/src/components/CardProfileInfo/index.jsx]: We don't leave the alt empty

File: frontend/src/components/CardProfileInfo/style.jsx [hunk 3]
```diff
@@ -1,22 +1,182 @@
...
+  .imgLinkedin {
+    right: 0.5rem;
+    top: 6rem;
+  }
+
+  .description {
+    text-align: center;
+  }
+  @media screen and (min-width: 500px) {
+    /* h1 {
```
Comment (LuigiMenez) [inline frontend/src/components/CardProfileInfo/style.jsx]: Dead code

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Some problems to solve

**Review by ghost — COMMENTED** [review]
Add alt and delete dead code

**Review by LauraCourquin — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**